data/operators/man_made/pipeline.json

```diff
@@ -2426,14 +2426,9 @@
       "displayName": "Kinder Morgan Texas Pipeline LLC",
       "id": "kindermorgantexaspipelinellc-0a386a",
       "locationSet": {"include": ["us"]},
-      "matchNames": [
-        "kinder morgan tejas pipeline",
-        "kinder morgan tejas pipeline llc"
```

Comment:
These apparently come from 599b859284b539a44cbd22996dde6bcc6b68b668, which removed two separate entries for Kinder Morgan Tejas Pipeline and Kinder Morgan Tejas Pipeline LLC. If that’s a separate company, I think the correct fix would be to restore one of the deleted entries. Based on your PR description, I’ve tagged [Kinder Morgan Tejas Pipeline (Q30973003)](https://www.wikidata.org/wiki/Q30973003) as “different from” Kinder Morgan Texas Pipeline LLC.

Reply:
Should be fixed